### PR TITLE
feature/v1-030/Page My Courses and components redesigned

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
In new design pages my courses and courses list look the same according to new mockup

### CODE REVIEW

File: frontend/src/components/Course/Course.tsx [hunk 1]
```diff
@@ -9,36 +9,38 @@ import {
...
   duration,
   lessons,
-  link,
+  styleProps,
   children,
 }) => (
-  <CourseContainer>
+  <CourseContainer size={styleProps?.size}>
     <ImageWrapper>
       <Image width={300} height={200} />
```
Comment (NikitaKhadnevich) [inline frontend/src/components/Course/Course.tsx:36]: We Will fix this magic number in future.

File: frontend/src/pages/MyCourses/MyCourses.tsx [hunk 2]
```diff
@@ -1,52 +1,72 @@
 import React from 'react';
 
 import { AuthorizedLayout } from 'components/Layout';
-
+import { PageContainer, CourseButton, CourseActions } from './styled';
```
Comment (NikitaKhadnevich) [inline frontend/src/pages/MyCourses/MyCourses.tsx:4]: `AuthorizedLayout` in `My Courses` as like `Corses` use `Grid` with 2 row?

**Review by NikitaKhadnevich — COMMENTED** [review]
Need some fix

File: frontend/src/pages/MyCourses/MyCourses.tsx (hunk 2 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/MyCourses/MyCourses.tsx:4]: My courses don't use grid

Comment (NikitaKhadnevich) [reply frontend/src/pages/MyCourses/MyCourses.tsx:4]: Ok, fine

File: frontend/src/components/Course/Course.tsx (hunk 1 above)
Comment (Julia-Maliutsina) [reply frontend/src/components/Course/Course.tsx:36]: removed

**Review by NikitaKhadnevich — APPROVED** [review]

File: frontend/src/pages/MyCourses/MyCourses.tsx [hunk 3]
```diff
@@ -1,52 +1,76 @@
...
+  imgWidth?: number;
+  imgHeight?: number;
+}
+
+const styleProps: CourseStyleTypes = {
+  fontSize: 14,
+  lineHeight: 18,
+  size: 'medium',
+  imgWidth: 300,
+  imgHeight: 200,
```
Comment (k-krasilnikova) [inline frontend/src/pages/MyCourses/MyCourses.tsx:51]: Move it please like primary or secondary style to the common component. Avoid sending a huge amount of props per tree. 
Confirm it with @hauteballe

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/pages/MyCourses/MyCourses.tsx (hunk 3 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/MyCourses/MyCourses.tsx]: Replaced it with 1 prop "size" that has 3 options: small, medium, large. All the parts of Course component are styled according to the chosen size.

**Review by hauteballe — APPROVED** [review]

**Review by hauteballe — APPROVED** [review]

**Review by NikitaKhadnevich — APPROVED** [review]

**Pull request closed without merging**